<?xml version="1.0"?>
<launch>
    <node pkg="velodyne_pointcloud" type="cloud_node" name="convert_veloscans">
	<param name="calibration" value="$(find ndt_fuser)/launch/velo32.yaml" />
    </node>
    <node pkg="ndt_mcl" type="3d_ndt_mcl_node" required="true" name="NDTMCL3D">
	<param name="points_topic" value="/velodyne_points" />
	<param name="odometry_topic" value="/vmc_navserver/odom" />
	<param name="map_resolution" value="0.4" />
	<!-- Specific sensor offset parameters for the example file -->
	<param name="set_sensor_pose" value="true" />
	<param name="sensor_pose_x" value="0.3" />
	<param name="sensor_pose_z" value="1.3" />
	<param name="sensor_pose_t" value="-1.62" />

	<param name="set_initial_pose" value="true" />
	<param name="pose_init_x" value="10.73" />
	<param name="pose_init_y" value="2.185" />
	<param name="pose_init_t" value="0.05" />
	
	<param name="load_map_from_file" value="true" />
	<param name="map_file_name" value="$(find ndt_fuser)/maps/basement_04m.jff" />
	<param name="save_output_map" value="false" />
	
     </node>   
</launch>
